<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QMainWindow" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>671</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1155</width>
    <height>962</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SpriteEditor3000</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121212;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="fpsVal">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="styleSheet">
        <string notr="true">background-color:#373737;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>221</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="eraseButton">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="eyeDropperButton">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Eye Dropper</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="colorButton">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="penButton">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Pen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="addFrame">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Add Frame</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="undo">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current Frame</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="currentFrameSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>531</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Draw Area</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="rotateRightButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);
</string>
       </property>
       <property name="text">
        <string>Rotate Right</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="rotateLeftButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(44, 44, 44);</string>
       </property>
       <property name="text">
        <string>Rotate Left</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="ClickableSpriteDisplay" name="drawLabel">
       <property name="maximumSize">
        <size>
         <width>508</width>
         <height>508</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="previewLabel">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Brush Color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="selectedColorBox" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 4px;
border: 2px solid white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="animationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(44, 44, 44);</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#373737;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_SpriteSheet"/>
    <addaction name="actionSave_SpriteSheet"/>
    <addaction name="actionLoad_SpriteSheet"/>
    <addaction name="actionExport_to_Gif"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_Right"/>
    <addaction name="actionNew_Frame"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Current_Frame"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPen_Tool"/>
    <addaction name="actionErase_Tool"/>
    <addaction name="actionEyedropper_Tool"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPreview_GIF"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_SpriteSheet">
   <property name="text">
    <string>New SpriteSheet</string>
   </property>
  </action>
  <action name="actionLoad_SpriteSheet">
   <property name="text">
    <string>Load SpriteSheet</string>
   </property>
  </action>
  <action name="actionExport_to_Gif">
   <property name="text">
    <string>Export to Gif</string>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="text">
    <string>Rotate Left</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="text">
    <string>Rotate Right</string>
   </property>
  </action>
  <action name="actionNew_Frame">
   <property name="text">
    <string>New Frame</string>
   </property>
  </action>
  <action name="actionDelete_Current_Frame">
   <property name="text">
    <string>Delete Current Frame</string>
   </property>
  </action>
  <action name="actionPen_Tool">
   <property name="text">
    <string>Pen Tool</string>
   </property>
  </action>
  <action name="actionErase_Tool">
   <property name="text">
    <string>Erase Tool</string>
   </property>
  </action>
  <action name="actionEyedropper_Tool">
   <property name="text">
    <string>Eyedropper Tool</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionSave_SpriteSheet">
   <property name="text">
    <string>Save SpriteSheet</string>
   </property>
  </action>
  <action name="actionPreview_GIF">
   <property name="text">
    <string>Preview GIF</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableSpriteDisplay</class>
   <extends>QLabel</extends>
   <header>clickablespritedisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
